plugins/modules/nae_ag.py

```diff
@@ -119,6 +152,8 @@ def main():
         apic_password=dict(type='str', default="", no_log=True),
         online=dict(type='bool', default=False),
         validate_certs=dict(type='bool', default=False),
+        run=dict(type='bool', default=None),
+        run_iter=dict(type='int', default=None),
```

Comment:
should we spell it out run_iterations to make it clearer?

Reply:
Sure